modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -148,4 +199,71 @@ private String getFullClassName(Element classElement) {
         }
         return className.toString() + classElement.getSimpleName();
     }
+
+    private String getMethodIdentifierString(Element methodElement) {
+        List<? extends TypeMirror> methodParams =
+            ((ExecutableType) methodElement.asType()).getParameterTypes();
```

Comment:
Rather than casting this to `ExecutableType`, I think we can call the `accept` method on the `TypeMirror` and pass a `TypeVisitor` that only handles `visitExecutable`